HttpServer.cs

```diff
@@ -231,10 +230,17 @@ public static void Create()
             }
         }
 
+        public static bool IsRunning()
```

Comment:
Is this used anywhere? Looks like maybe you intended to use it in Destroy()?

Reply:
It was left over from the BepInEx config, removed.